backend/Application/Services/AssetService.cs

```diff
@@ -100,4 +103,35 @@ public async Task<Response<GetListAssetsResponse>> GetListAsync(GetListAssetsReq
 
         return new Response<GetListAssetsResponse>(true, responseData);
     }
+
+    public async Task<Response<GetAssetResponse>> CreateAssetAsync(CreateAssetRequest requestModel)
+    {
+        var assetList = await _assetRepository.ListAsync();
+
```

Comment:
Check whether the category exists or not by categoryRepo. 
Because there is a case where no asset belongs to this category.